Offseason2022/src/main/java/frc/robot/commands/AutoShootDriveShoot.java

```diff
@@ -3,27 +3,73 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.AUTOConstants;
+import frc.robot.Constants.AUTOConstants.AutoTimer;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.FloorConveyor;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.TowerConveyor;
+import frc.robot.subsystems.Vision;
 
 /**
  *
  */
 public class AutoShootDriveShoot extends SequentialCommandGroup
 {
-  public AutoShootDriveShoot( )
+  private String m_pathname1 = AUTOConstants.kShootDriveShoot_path1;
+  private String m_pathname2 = AUTOConstants.kShootDriveShoot_path2;
+  private String m_pathname3 = AUTOConstants.kShootDriveShoot_path3;
+  private String m_pathname4 = AUTOConstants.kShootDriveShoot_path4;

```

Comment:
I don't think these names are necessary; I think you could just call the path names where they are needed, but we ca fix that later